components/resources/brave_components_strings.grd

```diff
@@ -1070,9 +1070,11 @@
       <message name="IDS_BRAVE_WALLET_BRAVE_REWARDS" desc="">Brave Rewards</message>
       <message name="IDS_BRAVE_WALLET_DOWNLOADING" desc="">Downloading Component...</message>
       <message name="IDS_BRAVE_WALLET_DISCLOSURE_CONFIRM" desc="">I understand</message>
-      <part file="speedreader_strings.grdp" />
-      <part file="ipfs_strings.grdp" />
+
       <part file="crypto_dot_com_strings.grdp" />
+      <part file="ipfs_strings.grdp" />
+      <part file="permissions_strings.grdp" />
+      <part file="speedreader_strings.grdp" />
```

Comment:
Made it in alphabetical order. Shouldn't break anything, but if someone thinks the other way, please say it.